presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenEvent.kt

```diff
@@ -4,13 +4,12 @@ import android.net.Uri
 import me.androidbox.component.agenda.AgendaActionType
 import me.androidbox.component.agenda.VisitorType
 import me.androidbox.component.event.VisitorInfo
-import me.androidbox.domain.authentication.model.Event
 
 sealed interface EventScreenEvent {
     data class OnPhotoUriAdded(val photoUri: Uri): EventScreenEvent
     /* Visitor Type i.e. All, Going, Not Going */
     data class OnSelectedVisitorType(val visitorType: VisitorType): EventScreenEvent
-    data class OnSaveEditOrDescriptionContent(val content: String): EventScreenEvent
+    data class OnSaveEditOrDescription(val title: String, val description: String): EventScreenEvent
```

Comment:
OnSaveEditOrDescription

You are missing the word "title" on the event right?

Reply:
@MKiperszmid Thanks for catching